### ISSUE TITLE
RegEx failing on IOS Edge/Safari

### ISSUE THREAD

**Issue by WynieCronje** [AzureAD/microsoft-authentication-library-for-js#5625]
### Core Library

MSAL.js v2 (@azure/msal-browser)

### Core Library Version

2.32.2

### Wrapper Library

MSAL Angular (@azure/msal-angular)

### Wrapper Library Version

2.5.2

### Public or Confidential Client?

Public

### Description

Website fails to log me in on my mobile device. When I check the logs on the server. It shows below exception.

### Error Message

`SyntaxError: Invalid regular expression: invalid group specifier name. `

After some investigarion I narrowed it down to this code.

```js
m.getCredentialType = function(c) {
    for (var a = g.Bv.CACHE_KEY_SEPARATOR, S = 0, N = Object.keys(g.d3); S < N.length; S++) {
        var re = N[S]
          , Me = g.d3[re].toLowerCase();
        if (-1 !== c.toLowerCase().search("(?<=" + a + "(https?:\\/\\/)?([\\w-]+\\.)*([\\w-]{1,63})(\\.(\\w{2,3}))(\\:[0-9]{4,5})?)" + a + Me + a))
            return g.d3[re]
    }
    return g.gT.NOT_DEFINED
}
```

Code in GitHub:
https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/c77682011de9cde8e141b8f404656bce3e3d62fb/lib/msal-common/src/cache/entities/CredentialEntity.ts#L110-L124

### Msal Logs

Not possible from mobile device

### MSAL Configuration

```javascript
{
}
```

### Relevant Code Snippets

```javascript
Happens when you call method `IPublicClientApplication.getAllAccounts();`
```

### Reproduction Steps

call getAllAccounts(); from an IOS device

### Expected Behavior

It should not throw a SyntaxError

### Identity Provider

Azure AD / MSA

### Browsers Affected (Select all that apply)

Edge, Safari

### Regression

@azure/msal-browser 2.31.0 | @azure/msal-angular 2.4.6

### Source

External (Customer)

**Comment by WynieCronje** [comment]
It seems like Safari does not support `lookbehind` regex.
https://stackoverflow.com/questions/51568821/works-in-chrome-but-breaks-in-safari-invalid-regular-expression-invalid-group

**Comment by WynieCronje** [comment]
I can see this issue was fixed in https://github.com/AzureAD/microsoft-authentication-library-for-js/commit/9160333d859beba44926833d74b73125ea84f526 When can we expect an update in `@azure/msal-angular` and `@azure/msal-browser` ?

**Comment by sameerag** [comment]
@WynieCronje This will be released in the first week of February as a part of the monthly release cadence for all MSAL JS libraries.

**Comment by gmantri** [comment]
@sameerag - Wondering if this got released in the latest version (@azure/msal-browser@2.33.0)? Did not see this being mentioned in the release notes, hence asking. Thanks.

**Comment by sameerag** [comment]
Yes it is. It is released as a part of [@azure/msal-common v9.1.0](https://github.com/AzureAD/microsoft-authentication-library-for-js/releases/tag/msal-common-v9.1.0), and since we also bumped `msal-common` to `10.0.0` [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/releases/tag/msal-common-v10.0.0), you do not see it in the latest.

**Comment by sameerag** [comment]
Closing since this is addressed.
